src/explore-education-statistics-admin/src/components/editable/Comments.tsx

```diff
@@ -133,21 +144,75 @@ const Comments = ({
 
         <ul className={styles.commentsContainer}>
           {orderBy(comments, ['created'], ['desc']).map(comment => {
-            const { createdBy } = comment;
+            const { createdBy, resolvedBy } = comment;
 
-            return (
+            return comment.resolved ? (
+              <li className="govuk-body-s">
+                <p className="govuk-!-margin-0">
+                  <strong>Comment resolved</strong>
+                </p>
+                <p className="govuk-!-margin-0">
+                  <FormattedDate format="dd/MM/yy HH:mm">
+                    {comment.resolved}
+                  </FormattedDate>
+                </p>
+                <p className="govuk-!-margin-0">
+                  {`by ${resolvedBy?.firstName} ${resolvedBy?.lastName}`}
+                </p>
+                <Details className="govuk-!-margin-0" summary="See comment">
+                  <p className="govuk-!-margin-0">
+                    <strong>
+                      {`${createdBy.firstName} ${createdBy.lastName}`}
+                    </strong>
+                  </p>
+                  <p className="govuk-!-margin-0">
+                    {'Created: '}
+                    <FormattedDate format="dd/MM/yy HH:mm">
+                      {comment.created}
+                    </FormattedDate>
+                  </p>
+                  {comment.updated && (
+                    <p className="govuk-!-margin-0">
+                      {'Updated: '}
+                      <FormattedDate format="dd/MM/yy HH:mm">
+                        {comment.updated}
+                      </FormattedDate>
+                    </p>
+                  )}
```

Comment:
This is a little chaotic tbh. Think we could refactor a lot of this to use `dl/dt/dd` elements.

The markup would be more semantically correct and would read nicer. We'll probably need some extra styling in `Comments.module.scss`, but we can do away with all the `strong` elements and `govuk-!-margin-0`.